### PR TITLE
context folder added on frontend.

### PR DESCRIPTION
**Pull request by OLINYA-PAUL**
I have added another folder called context, it will help manage small state management for components on frontend.

### CODE REVIEW

File: projectSpark-frontend/README.md
Comment (freedompraise) [inline projectSpark-frontend/README.md:1]: Please can you revert the changes you made on the readme. 
The double asterikes put on names was so as to highlight them

File: projectSpark-frontend/components/Navber.jsx
Comment (freedompraise) [inline projectSpark-frontend/components/Navber.jsx:1]: can you rename this file to "navbar". That's the right spelling

File: projectSpark-frontend/dashboard/dashboard.jsx
Comment (freedompraise) [inline projectSpark-frontend/dashboard/dashboard.jsx:1]: can you verify the folder location for this?

File: projectSpark-frontend/README.md
Comment (Paul-Rust) [reply projectSpark-frontend/README.md:1]: ok

File: projectSpark-frontend/components/Navber.jsx
Comment (Paul-Rust) [reply projectSpark-frontend/components/Navber.jsx:1]: yeah done

File: projectSpark-frontend/dashboard/dashboard.jsx
Comment (Paul-Rust) [reply projectSpark-frontend/dashboard/dashboard.jsx:1]: the location has been verified

**Comment by freedompraise** [comment]
Hey @OLINYA-PAUL . I noticed you closed both pr's.
What's the plan?
